### PR TITLE
Menu component

### PR DESCRIPTION
**Pull request by yetieaterxb1**
-Built and styled front end
-Wrote tests
-Simplified seed script

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -2,5 +2,9 @@
   "presets": [
     "@babel/env",
     "@babel/react"
-  ]
+  ],
+    "plugins": [
```
Comment (merlinv12) [inline .babelrc]: whats this plugin for? The airbnb linter?

File: client/src/components/modal.jsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      position: relative;
+      padding: 20px
+  }
+`;
+
+const modal = props => (
+  <Modal>
+    <div
+      className="modal-wrapper"
+      style={{
```
Comment (merlinv12) [inline client/src/components/modal.jsx]: Would it be possible to add this inline style to the style component CSS you have starting on line 6.

File: .babelrc (hunk 1 above)
Comment (yetieaterxb1) [reply .babelrc]: I believe it used for only using the 'helpers' that are needed during the build. I havent used it yet and will remove if I dont.

File: client/src/components/modal.jsx (hunk 2 above)
Comment (yetieaterxb1) [reply client/src/components/modal.jsx]: I tried moving it but ran into issues with the css loading.

File: .gitignore [hunk 3]
```diff
@@ -1 +1,2 @@
 node_modules
+client/dist/bundle.js
```
Comment (merlinv12) [inline .gitignore]: Add the .DS_Store to gitignore.

**Pull request merged**